src/src/thread/safe/list/hillel/Demo.java

```diff
@@ -0,0 +1,14 @@
+package thread.safe.list.hillel;
+
+public class Demo {
+    public static void main(String[] args) {
+        ThreadSafeList<Integer> list = new ThreadSafeList<>();
+        list.add(1);
+        list.add(2);
+        list.add(3);
```

Comment:
If it is a thread-safe list I expect it to be tested using multiple threads, 10 for example.